src/containers/Bookings.js

```diff
@@ -1,19 +1,19 @@
 import React, { Component } from 'react';
 import Search from '../components/Search.js';
-// import Results from '../components/Results.js';
-// import FakeBookings from '../data/fakeBookings.json';
+import Result from '../components/Result.js';
+import FakeBookings from '../data/fakeBookings.json';
 
 export default class Bookings extends Component {
   search = () => {
     console.info('TO DO!');
   };
-  
+
   render() {
     return (
       <div className="App-content">
         <div className="container">
           <Search search={this.search} />
-          {/* <Results results={this.state.results} /> */}
+          <Result booking={FakeBookings} />
```

Comment:
Great job on passing in the in the json file in the `booking` prop for the `Result` component :)